x/cronos/types/messages.go

```diff
@@ -335,18 +335,25 @@ func NewMsgStoreBlockList(from string, blob []byte) *MsgStoreBlockList {
 
 var errDummyIdentity = stderrors.New("dummy")
 
+const MaximumBlobLength = 20480
```

Comment:
is it long enough?

Reply:
20kb should be fine right?

Reply:
not sure, we have the max tx size limit anyway